pyrasterframes/src/main/python/docs/numpy-pandas.pymd

```diff
@@ -32,15 +32,15 @@ t.cells.shape, t.cells.nbytes
 
 As discussed in the @ref:[raster writing chapter](raster-write.md#dataframe-samples), a pretty display of Pandas DataFrame containing _tiles_ is available by importing the `rf_ipython` submodule. In addition, as discussed in the @ref:[vector data chapter](vector-data.md), any geometry type in the Spark DataFrame will be converted into a Shapely geometry. Taken together, we can easily get the spatial information and raster data as a NumPy array, all within a Pandas DataFrame.
 
-```python spark_session,  echo=False
+```python, spark_session,  echo=False
```

Comment:
this page could use a little rethinking. The user should prefer to `display(df)` than to do `display(df.toPandas)` which is how it's still written.